src/GoBoard.d.ts

```diff
@@ -0,0 +1,75 @@
+export type Vertex = [number, number];
```

Comment:
Please follow code style of the other files: No semicolons.

Reply:
Sure